src/utils/__tests__/checkError.test.ts

```diff
@@ -40,6 +42,7 @@ describe("checkError", () => {
 
   it("should display the message with detail as an array", () => {
     const errorMsg = checkError({
+      // @ts-ignore
```

Comment:
Why? :(

Reply:
Added a comment to the ignore. The defined response doesn't satisfy the AxiosError type. This could be improved later but I think it's fine to first pass the CI check.